shaders/Crystal.gdshader

```diff
@@ -50,5 +50,5 @@ void fragment() {
     METALLIC = metallic;
     SPECULAR = specular;
 
-    EMISSION = clamp(((dissolveOutline + emission.rgb) * roughnessUV * emissionEnergy), 0.0f, dissolveValue);
+    EMISSION = emission.rgb * roughnessUV * emissionEnergy;
```

Comment:
I assume it is intentional that emission is no longer clamped based on the dissolve value.